### PR TITLE
added Idea Xchange to React projects list

### PR DESCRIPTION
**Pull request by apoorvdwi**
## What does this PR do?

Adds the Idea Xchange project to "built with Appwrite" React projects list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,30 +3,34 @@
...
     - [Golang](#golang)
+    - [Preact](#preact)
     - [Python](#python)
     - [React](#react)
+    - [React Native](#react-native)
     - [Svelte](#svelte)
-    - [Unity](#unity)
     - [Vue](#vue)
-    - [Nuxt](#nuxt)
+    - [Unity](#unity)
```
Comment (adityaoberai) [inline README.md:33]: @apoorvdwi could you please revert the changes made to the Table of Contents?

**Review by adityaoberai — CHANGES_REQUESTED** [review]
@apoorvdwi please make the request change. The PR is good to go after that.

File: README.md (hunk 1 above)
Comment (apoorvdwi) [reply README.md]: Done

**Comment by adityaoberai** [comment]
Thanks @apoorvdwi!

I was just reviewing the project repository. Would it be possible for you to update your project's documentation in the main Readme? It would definitely add a lot more context about the project as well as assist anyone who intends to contribute to the project.

Additionally, the `appwrite/` folder in your repo contains the `docker-compose.yml` file needed to set up an Appwrite instance but lacks the `.env` file, which is also very necessary.

**Comment by adityaoberai** [comment]
Hi @apoorvdwi 
Did you get a chance to look at the recommended changes?

**Comment by apoorvdwi** [comment]
Hey @adityaoberai, I've made the requested changes.
1. updated the readme file
2. added env in appwrite folder

**Review by adityaoberai — APPROVED** [review]
Looks good, @apoorvdwi!

Merging the PR now
